packages/capesolo.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,8 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+$toolName = 'CAPEsolo'
+$category = 'Debuggers'
+$version = '==0.4.23'
+
+VM-Install-With-Pip -toolName $toolName -category $category -version $version
```

Comment:
```suggestion
VM-Install-With-Pip -toolName $toolName -category $category -version $version -arguments ""
```

@sara-rn are you planing to add this change to this PR? :smile:

Reply:
@Ana06 thanks for the suggestion!